client/blocks/login/style.scss

```diff
@@ -160,8 +160,9 @@
 }
 
 .login__form-signup-link {
+	clear: both;
 	font-size: 13px;
-	margin-top: 20px;
+	padding-top: 20px;
```

Comment:
This is to fix a wrong margin applied for regular OAuth clients:

![screenshot](https://user-images.githubusercontent.com/594356/63116264-8263f080-bf99-11e9-8254-27d25cf10a56.png)